app/views/application_drafts/_student_fields.html.slim

```diff
@@ -35,12 +35,10 @@
       = student_field_error_hint(index, application_draft, :application_coding_level)
   = s.input :application_community_engagement, label: 'Tell us about your community involvement.', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_community_engagement) }, hint: (student_field_error_hint(index, application_draft, :application_community_engagement) || 'When did you attend your first Rails Girls, Railsbridge, Black Girls Code, PyLadies, or similar workshop? Have you been involved in organization/coaching/study groups since then? Tell us all about it!')
   .form-inline
-    = s.input :application_learning_period, :collection => User::MONTHS_LEARNING, include_blank: true, label: 'For how many months have you actively been learning after your first workshop?', disabled: !may_edit?(s.object), wrapper_html: { class: student_field_error_class(index, application_draft, :application_learning_period)}, hint: student_field_error_hint(index, application_draft, :application_learning_period)
```

Comment:
I think the `User::MONTHS_LEARNING` is no longer needed now. Could you verify that in your branch and remove it from the `User` model as well?